format_nomad_with_env.sh

```diff
@@ -167,6 +167,19 @@ export_log_conf (){
 
 # This actually performs the templating using Perl's regex engine.
 # Perl magic found here: https://stackoverflow.com/a/2916159/6095378
+export_log_conf (){
+    export LOGGING_CONFIG="
+        logging {
+          type = \"awslogs\"
+          config {
+            awslogs-region = \"$REGION\",
+            awslogs-group = \"data-refinery-log-group-$USER-$STAGE\",
+            awslogs-stream = \"log-stream-$1-docker-$USER-$STAGE\"
+          }
+        }
+"
+}
```

Comment:
Are you sure you meant to add this? It seems like the block of code immediately preceding this already does this.